collectfast/tests/command/test_command.py

```diff
@@ -20,16 +20,23 @@
         COLLECTFAST_STRATEGY="collectfast.strategies.boto3.Boto3Strategy",
     ),
 }
+filesystem_backend_confs = {
+    "filesystem": override_django_settings(
+        STATICFILES_STORAGE="django.core.files.storage.FileSystemStorage",
+        COLLECTFAST_STRATEGY="collectfast.strategies.filesystem.FileSystemStrategy",
+    ),
+    "cachingfilesystem": override_django_settings(
+        STATICFILES_STORAGE="django.core.files.storage.FileSystemStorage",
+        COLLECTFAST_STRATEGY="collectfast.strategies.filesystem.CachingFileSystemStrategy",
+    ),
+}
```

Comment:
Unless we're using `filesystem_backend_confs` for something other than defining `all_backend_confs`, I'd prefer to move the definitions back into `all_backend_confs` and removing `filesystem_backend_confs`.

Reply:
Ok.  I have some tests that will need to run against filesystem backends in the next issue but I can add it there.